python-sdk/src/astro/sql/operators/cleanup.py

```diff
@@ -55,7 +55,7 @@ class CleanupOperator(AstroSQLBaseOperator):
         Very relevant if using a synchronous executor. The default is 3.
     :param retry_delay: Delay between running retries. Very relevant if using a synchronous executor.
         The default is 10s.
-    :param run_sync_mode: Whether to wait for the DAG to finish or not. Set to False if you want
+    :param run_sync_mode: Whether to wait for the DAG to finish or not. Set to True if you want
         to immediately clean all DAGs. Note that if you supply anything to `tables_to_cleanup`
```

Comment:
is `Set to True if you want to immediately clean all DAGs.` right? or it should be `tables`?

Reply:
Makes sense

Reply:
```suggestion
        to immediately clean all tables. Note that if you supply anything to `tables_to_cleanup`
```